src/components/Local.vue

```diff
@@ -73,56 +182,95 @@ export default {
 
   data: function() {
     return {
-      stringCopied: "Copied to clipboard",
-      newClip: "",
-      clips: [],
-      preview: "Here you'll see preview as soon as you start typing.",
+      // messeges to be shown
+      stringCopied: "Copied to clipboard.",
+      preview: "Start typing to see preview.",
+      emptyStringText: "Cannot add empty clip.",
+      stringNotCopied:
+        "Could not copy to the clipboard, is the App permitted ?",
+      gotDeletedString: "Clip deleted!",
+      messageAddToFav: "Added to favourites!",
+      messageArchived: "Clip Archived!",
+
+      // logical parameters
       emptyString: false,
       copied: false,
       notCopied: false,
-      emptyStringText: "Cannot add empty string",
-      timeout: 3000,
-      stringNotCopied:
-        "Could not copy to the clipboard, is the App permitted ?",
-      gotDeletedString: "Clip has been successfully deleted",
-      just_got_deleted: false
+      justGotDeleted: false,
+      showArchived: false,
+      justGotArchived: false,
+
+      // logic for the new life
+      newClip: "",
+      clips: [],
+
+      // forein component requirements
+      timeout: 1500
     };
   },
 
   mounted() {
     if (JSON.parse(this.$cookies.get("clips"))) {
       this.clips = JSON.parse(this.$cookies.get("clips"));
     }
-    this.clips.forEach(e => {
-      console.log(e.id, e.title);
+
+    // adding sample clip
+    this.clips.push({
+      id: 0,
+      title: "Sample clip",
+      fav: false,
+      created_on: this.getNow(),
+      archived: false
     });
```

Comment:
I think we need to wrap this in an if statement which checks if the length of clips is 0.